tox.ini

```diff
@@ -14,8 +14,16 @@ usedevelop = true
 basepython =
     python2.7
 commands =
-    coverage run -m zope.testrunner --test-path=src []
+    coverage run -m zope.testrunner --test-path=src --auto-color -vv []
```

Comment:
Perhaps make it `coverage run -m zope.testrunner --test-path=src {posargs:--auto-color -vv}`, so people can override?